ext/background/api.js

```diff
@@ -233,13 +237,11 @@ async function request(path, params = {}) {
 
   if (resp.ok) return resp.json();
 
-  if (resp.status === 401 || resp.status === 403) {
-    if (isLogin) {
-      console.log('failed login attempt, likely invalid cookies...');
-    } else {
-      console.log('no longer authenticated, refreshing sessions...');
-      await refreshSessions(true);
-    }
+  if (isLogin) {
+    console.log('failed login attempt, likely invalid cookies');
+  } else if (resp.status === 401 || resp.status === 403) {
+    console.log('no longer authenticated, refreshing sessions');
+    await refreshSessions(true);
```

Comment:
Do we still need this change from before, or could we revert it? I might be mis-reading it, but it seems like the effect of this will be to change what's logged for login requests, so that we no longer log status or status text for login errors other than 401/403. But it seems like that extra debug info might still be useful?

Reply:
Good point, reverted.